src/components/Portal/Portal.tsx

```diff
@@ -175,13 +176,12 @@ class Portal extends AutoControlledComponent<ReactPropsStrict<PortalProps>, Port
   }
 
   private handlePortalRef = (portalNode: HTMLElement) => {
-    this.portalNode = portalNode
+    handleRef(this.portalRef, portalNode)
   }
 
   private handleTriggerRef = (triggerNode: HTMLElement) => {
-    this.triggerNode = triggerNode
-
-    _.invoke(this.props, 'triggerRef', triggerNode)
+    handleRef(this.triggerRef, triggerNode)
```

Comment:
once again this strikes my cognitive process :) I really think that `setRef` will be much more intuitive name for this method as, essentially, this is what it does